test/core/storage/car/CMakeLists.txt

```diff
@@ -11,6 +11,13 @@ target_link_libraries(car_test
     ipfs_datastore_in_memory
     unixfs
     )
-target_include_directories(car_test PUBLIC
```

Comment:
make car_test fails now

Reply:
Works on my machine and on CI

Reply:
it works because file is created in another task